plenty_api/api.py

```diff
@@ -824,7 +825,69 @@ def plenty_api_set_image_availability(self,
                                              path=path,
                                              data=data)
 
-        if not response:
-            return False
+        return response
```

Comment:
I agree that this method is not optimal but this change feels out of place. Especially because the other return paths still return a boolean and the method description still declares the return value to be a boolean.

Reply:
This is related to a change in `__plenty_api_request` which returns the error response instead of `None` when a request fails. I changed this because I need the error message to forward it to the end user, but the focus was on getting it to work quickly.
What do you think is the best way we should be forwarding error messages? Exceptions might be cleaner, but would require more code to catch them when using the API.

Reply:
I guess the best course of action, is to rewrite that method as it doesn't align with the rest of the library. I will do that after merging this patchset and before publishing v.0.2.8